_data/events/frankfurtAmMain.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "GDCR 17 in Frankfurt am Main",
+  "url": "https://www.eventbrite.de/e/global-day-of-coderetreat-2017-in-frankfurt-tickets-37800982757",
+  "moderators": [
+    "Muhammad Ali Kazmi",
```

Comment:
Please only list either your twitter handle or your full name (preferably the twitter handle(